src/backend/opencl/generators/ocl_generic_rx_generator.cpp

```diff
@@ -63,6 +64,7 @@ bool ocl_generic_rx_generator(const OclDevice &device, const Algorithm &algorith
     case OclDevice::Navi_10:
     case OclDevice::Navi_12:
     case OclDevice::Navi_14:
+    case OclDevice::Navi_21:
```

Comment:
Navi 21 is RDNA 2.0 and I don't remember writing any asm code for it. It probably shouldn't use asm.

Reply:
Unless you can test it on RDNA 2.0 GPU and confirm that it works :)

Reply:
Split to separate logic which sets `isNavi` without setting `isAsm`.

I think someone had an Apple W6xxx card and did test it and it did work, but Apple gets no asm anyway being generic OpenCL... so setting isAsm didn't really matter?